### PR TITLE
Part 1: docs styleguide polishing

### PR DESCRIPTION
**Pull request by s-makin**
## Proposed Commit Message
```
Part 1 polishing (tutorials/how-tos)

- Tutorial and How-to landing pages have been revised/expanded.
- All tutorial/how-to pages have been "polished".
- Headings in the nav bar have been shortened for readability.
- All pages specifically ordered (no longer "random"). See below.

Goals here were improving navigation, readability, and
standardising.

Polishing = making sure pages conform to the Ubuntu style guide. 
Also includes minor language edits for clarity or to avoid word
repetitions, etc. Changed from passive to active, where possible.

All "notes" of all kinds have been standardised to the most
prevalent style used, which was the MyST Markdown format. I will 
add the various options for this to our style guide (current WIP).

The headings were good (descriptive :+1:) but a bit too long due to
repetition of "how to ...", so I have standardised these too. The
nav bar now has all pages organised in a particular order, and no
longer alphabetical by page name. I have grouped all pages by what
command they cover, and organised these commands alphabetically, 
which I think makes more sense.

If I've made any errors, lmk - but should be straightforward since
it was already mostly standardised.
```

### CODE REVIEW

File: docs/howtoguides/enable_realtime_kernel.md [hunk 1]
```diff
@@ -1,21 +1,20 @@
 # How to enable Real-Time Kernel
 
 ```{caution}
-Real-Time Kernel is currently in beta.
+Real-Time Kernel is currently in beta and is only supported on 22.04.
```
Comment (orndorffgrant) [inline docs/howtoguides/enable_realtime_kernel.md:4]: Oops, this is no longer beta from our perspective. You can also remove all instances of "--beta" in this file.
```suggestion
Real-Time Kernel is only supported on 22.04.
```

File: docs/howtoguides/how_to_collect_logs.md [hunk 2]
```diff
@@ -1,21 +1,27 @@
...
+problems with `pro`.
+
 It puts together:
-* The Ubuntu Pro Client configuration file (the default is `/etc/ubuntu-advantage/uaclient.conf`)
+
+* The Ubuntu Pro Client configuration file (the default is
+  `/etc/ubuntu-advantage/uaclient.conf`)
 * The Ubuntu Pro Client log files (the default is `/var/log/ubuntu-advantage*`)
-* The files in `/etc/apt/sources.list.d/*` related to UA
+* The files in `/etc/apt/sources.list.d/*` related to Ubuntu Advantage
```
Comment (orndorffgrant) [inline docs/howtoguides/how_to_collect_logs.md:18]:
```suggestion
* The files in `/etc/apt/sources.list.d/*` related to Ubuntu Pro
```

**Review by orndorffgrant — APPROVED** [review]
Really really great! This elevates our documentation a lot :) 
Just left a couple tiny nitpicks

File: docs/howtoguides/enable_fips.md [hunk 3]
```diff
@@ -1,32 +1,41 @@
...
+Alternatively, you can enable FIPS using the Ubuntu Pro Client, which will
+install a FIPS-certified kernel and core security-related packages such as
+`openssh-server/client` and `libssl`. 
 
-Alternatively, enable FIPS using the Ubuntu Pro Client will install a FIPS-certified kernel and core security-related
-packages such as openssh-server/client and libssl. Note: disabling FIPS on an image is not yet
-supported
+```{note}
+Note that disabling FIPS on an image is not yet supported.
+```
```
[the remark this answers is not in the document]
Comment (s-makin) [reply docs/howtoguides/enable_fips.md:18]: Good shout - I've now also put that danger admonition above the other one. On reflection, I think it's a more logical order to discuss them in.

File: docs/howtoguides/enable_realtime_kernel.md (hunk 1 above)
Comment (s-makin) [reply docs/howtoguides/enable_realtime_kernel.md:4]: Good to know! I've removed them all now

File: docs/howtoguides/how_to_collect_logs.md (hunk 2 above)
Comment (s-makin) [reply docs/howtoguides/how_to_collect_logs.md:18]: Good catch! Thanks Grant :) I'm still a bit woolly on exactly what the naming convention currently is, but that's cleared it up

**Pull request merged**